Examples/SwiftUIExample/SwiftUIExample/SceneDelegate.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SceneDelegate.swift
+//  SwiftUIExample
+//
+//  Created by Tom on 7/4/24.
+//
+
+import UIKit
+import BezierSwift
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+  
+  var window: UIWindow?
+  
+  func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    guard let windowScene = scene as? UIWindowScene else { return }
+    
+    self.window = windowScene.windows.first { $0.isKeyWindow }
```

Comment:
```suggestion
    self.window = UIWindow(windowScene: windowScene)
```
예제 코드이지만 windowScene 생성자를 사용하면 좋을 것 같아요

```swift
class SceneDelegate: UIResponder, UIWindowSceneDelegate {
  weak var windowScene: UIWindowScene?
  // ...
}

extension SceneDelegate: BezierSwiftDelegate {
  func windowsForThemeUpdate() -> [UIWindow] {
    self.windowScene?.windows ?? [] // or key window
  }
}
```

Reply:
감사합니다. 아래 코드로 변경하겠습니다!